react/week2/to-do-app/src/ToDo.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+
+function ToDo(props){
+    const [enabled, setEnabled] = useState(false);
+    const handleChange = (id) => {
+        setEnabled(prevState => !prevState);
+        const style=enabled?'none':'line-through';
+        const labelElement=document.getElementById('label'+id);
+        if(labelElement){
+            labelElement.style.textDecorationLine=style;
+        }
+      }
+
+    return(
+      <div>
+        <li><label id={"label" + props.id} style={{ textDecorationLine: 'none' }}>
```

Comment:
```suggestion
        <li>
          <label id={"label" + props.id} style={{ textDecorationLine: enabled ? 'none' : 'line-through'  }}>
```

Reply:
Since we have access to the state variable inside our JSX, we can use conditionals directly in our styling so there's no need to first select the element through document.getElementById.

Reply:
Thank you, i have resolved this part of code